docs/error-reporting/platform-integrations/source-map.md

```diff
@@ -9,88 +9,310 @@ import Tabs from '@theme/Tabs';
+  "backtrace:upload": "npx --package @backtrace/javascript-cli backtrace-js upload OUTPUT_DIRECTORY UPLOAD_OPTIONS"
 }
 ```
 
-You need to replace `<universe>` with your universe name, and `<token>` with the symbol upload token generated in your Project Settings under the Symbols section. For on-premise Backtrace servers, replace `<universe>.sp.backtrace.io:6098` with the server and port of your Backtrace instance.
+Ensure to replace `OUTPUT_DIRECTORY` with the path to the directory where your transpiled scripts are stored.
 
-## Step 4: Set Up Automatic Sourcemap Uploads
+Replace `UPLOAD_OPTIONS` with either `--subdomain <your subdomain> --token <your token>` or `--url <your upload URL>`, obtained from [the symbol submission URL](#obtaining-symbol-submission-url).
 
-This method uploads all artifact .map files.
+#### Configuration File
 
-First, append `npm run upload` to the `build` command in your package.json's `scripts` section:
+Instead of providing options in script argument lines, you can configure them in the `.backtraceclirc` configuration file:
 
-```json
-"scripts": {
-  "build": "my current build command && npm run upload",
-  ...
+```jsonc
+{
+  "path": "OUTPUT_DIRECTORY",
+  "upload": {
+    "subdomain": "your subdomain",
+    "token": "your token",
+    // or
+    "url": "your upload URL"
+  }
 }
 ```
 
-Finally, add the action script `upload` to run `backtrace-sourcemap` with `package.json` and the build folder as parameters. In this example, both the `package.json` and the `dist` folder are assumed to be located in the project root directory:
+For more details consult `@backtrace/javascript-cli` README.
+
+### Step 3: Set Up Automatic Processing and Upload
+
+To process and upload your artifacts, you must first run `npm run backtrace:process` and then `npm run backtrace:upload`.
 
+To ensure that this is done automatically, you can add these commands to your production script:
 ```json
 "scripts": {
-  "build": "my current build command && npm run upload",
-  "upload": "./node_modules/.bin/backtrace-sourcemap upload package.json dist"
+  "build": "my current build command && npm run backtrace:process && npm run backtrace:upload"
 }
 ```
+## Project Bundler
 
-## Step 5: Set Up Automatic UUID Generation in `backtrace-node` or `backtrace-js`
+<Tabs
+groupId="project-bundler"
+defaultValue="webpack"
+values={[
+{label: 'Webpack', value: 'webpack'},
+{label: 'Rollup', value: 'rollup'},
+{label: 'Vite', value: 'vite'},
+]}>
 
-To match a sourcemap file with an error report, a UUID is generated and attached to both the sourcemap file and the outgoing crash report.
+<TabItem value="webpack">
 
-In your Backtrace client initialization code, call `client.setSymbolication()` to enable automatic UUID generation:
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request GET 'https://api.us-west-1.saucelabs.com/api-testing/rest/v4/<hookId>' | json_pp
```

Comment:
@perf2711 - Are these lines part of the webpack tab? Or are they generic to all bundlers?